eng/.docsettings.yml

```diff
@@ -26,7 +26,7 @@ language: net
 root_check_enabled: True
 
 required_readme_sections:
-  - "Azure .+ client library for .NET"
+  - "Azure (.+ client library for .NET|AI Agent Server Adapter for .*.NET)"
```

Comment:
We can zap this since we're not going to have a full README and need L149-151 with exceptions.


FOLLOW-UP:   Issue needed.